includes/shortcodes.php

```diff
@@ -596,6 +596,15 @@ function give_totals_shortcode( $atts ) {
 	// Total Goal.
 	$total_goal = give_maybe_sanitize_amount( $atts['total_goal'] );
 
+	/**
+	 * Give Action fire before the shortcode is rendering is started.
+	 *
+	 * @since 2.1.4
+	 *
+	 * @param array $atts shortcode attribute.
+	 */
+	do_action( 'give_totals_goal_shortcode_start', $atts );
```

Comment:
@raftaar1191  rename it to `give_totals_goal_shortcode_before_render`

Reply:
Changed @ravinderk